packages/format-currency/src/index.ts

```diff
@@ -194,7 +205,11 @@ export function formatCurrency(
 ): string {
 	const locale = options.locale ?? defaultLocale ?? getLocaleFromBrowser();
 	if ( ! doesCurrencyExist( code ) ) {
-		code = 'USD';
+		// eslint-disable-next-line no-console
+		console.warn(
+			`formatCurrency was called with a non-existent currency "${ code }"; falling back to ${ fallbackCurrency }`
+		);
+		code = fallbackCurrency;
```

Comment:
This part seems a bit repetitive, does it make sense to abstract it to a helper function?

Reply:
Good idea.